vizro-ai/hatch.toml

```diff
@@ -48,6 +48,7 @@ prep-release = [
 ]
 pypath = "hatch run python -c 'import sys; print(sys.executable)'"
 test = "pytest tests {args}"
+test-e2e-vizro-ai-ui = "pytest -vs --reruns 1 tests/e2e/test_vizro_ai_ui.py --headless {args}"
```

Comment:
If we add this as a hatch script, does it run locally if I was to use that command?

Reply:
Yes, anyway to use it you need to run the VIzroAI UI first